src/send/SendConfirmation.test.tsx

```diff
@@ -207,10 +207,10 @@ describe('SendConfirmation', () => {
     const { getByTestId } = renderScreen({}, mockScreenPropsWithPreparedTx)
 
     const feeComponent = getByTestId('LineItemRow/SendConfirmation/fee')
-    expect(getElementText(feeComponent)).toEqual('0.004 CELO')
+    expect(getElementText(feeComponent)).toEqual('0.01 CELO')
 
     const totalComponent = getByTestId('TotalLineItem/Total')
-    expect(getElementText(totalComponent)).toEqual('~1.02 cUSD')
+    expect(getElementText(totalComponent)).toEqual('~1.05 cUSD')
```

Comment:
this only shows 2 decimals currently if the amount is >= 0.01, so if fees is <0.01, they won't show up here. Checking with product on how many decimals we want to show here and will make a follow up issue if this needs to be changed. (Needs some refactor to the TokenDisplay component)

Reply:
is that the case even if the fee is in ETH? like we don't show ETH fees if they're less than 0.01 ETH?